docs/Using-Fleet/Adding-hosts.md

```diff
@@ -206,4 +206,51 @@ installation should appear as the same host in the Fleet UI. If other settings a
 entries will appear in the Fleet UI. The older entries can be automatically cleaned up with the host
 expiration functionality configured in the application settings (UI or fleetctl).
 
+## Grant full disk access to osquery on macOS
+macOS does not allow applications to access all system files by default. If you are using MDM, you
+can deploy a "Privacy Preferences Policy Control" policy to grant Orbit or osquery that level of
+access. This is necessary to query for files located in protected paths as well as to use event
+tables that require access to the [EndpointSecurity
+API](https://developer.apple.com/documentation/endpointsecurity#overview), such as *es_process_events*.
+
+### Creating the configuration profile
+#### Obtaining identifiers
+If you use plain osquery, instructions are [available here](https://osquery.readthedocs.io/en/stable/deployment/process-auditing/).
+
+On a system with osquery installed via the Fleet osquery installer (Orbit), obtain the
+`CodeRequirement` of it by running:
+
+`codesign -dr - /private/var/lib/orbit/bin/orbit/macos/edge/orbit`
+
+The output should be similar or identical to: 
+
+`Executable=/private/var/lib/orbit/bin/orbit/macos/edge/orbit
```

Comment:
I feel like we need to add something to the style guide about inline code vs code blocks. Most places we're using inline code in the text, but code blocks for standalone code snippets:

To run the preview using `fleetctl`, use the following command:

```
fleetctl preview
```

Not sure this really needs changing here, but likely something we need to nail down. 

@DominusKelvin, @mike-j-thomas Thoughts?

Reply:
Nice catch Kathy. So my thinking(and this is how I use it as well). In-line code is mostly for like a variable name or a code identifier of some sort. 

An outright code block is then used for a block of code.

Reply:
Is it better like this? I left it as in-line for the commands, but then code block for the output.

Reply:
Created a new issue for this. Since it seems like we're mostly using them the way I mentioned in my original comment, we'll go with that. @GuillaumeRoss, I'm making the edits for that now.